lua/gatekeeper_config.lua

```diff
@@ -0,0 +1,22 @@
+local ffi = require("ffi")
+local gatekeeperc = require("gatekeeperc")
+
+-- A table that contains all the required functional blocks.
+local blocks = {}
+
+-- TODO Add configuration for other functional blocks.
+local block_names = {}
+
+function gatekeeper_init()
+	local ret = 0
+
+	for key, value in ipairs(block_names) do
+		blocks[#blocks + 1] = require(value)
```

Comment:
Why doesn't a block initialize itself? It would avoid the next loop, and wouldn't  really change what happens here.